nitro-protocol/gas-benchmarks/benchmark.test.ts

```diff
@@ -199,72 +205,100 @@ describe('Consumes the expected gas for sad-path exits', () => {
     ).toEqual(gasRequiredTo.ETHexitSadLedgerFunded.satp.total);
   });
 
-  // TODO unskip this test when the contracts are satp compatible
-  it.skip(`when exiting a virtual funded (with ETH) channel`, async () => {
+  it(`when exiting a virtual funded (with ETH) channel`, async () => {
     // begin setup
     await (
-      await nitroAdjudicator.deposit(MAGIC_ADDRESS_INDICATING_ETH, LforJ.channelId, 0, 10, {
-        value: 10,
-      })
+      await nitroAdjudicator.deposit(
+        // This deposit represents what in reality would likely be two deposits (one from Bob, one from Ingrid)
+        MAGIC_ADDRESS_INDICATING_ETH,
+        LforV.channelId,
+        0,
+        amountForAliceAndBob,
+        {
+          value: amountForAliceAndBob,
+        }
+      )
     ).wait();
     // end setup
-    // initially                   ⬛ ->  L  ->  J  ->  X  -> 👩
+    // initially                   ⬛ ->  L  ->  V  -> 👨
```

Comment:
Not entirely sure what the face emoji in the comment means. Does the virtual channel only allocate to one participant?

Reply:
I added some explanation here [c5ede49](https://github.com/statechannels/go-nitro/pull/831/commits/c5ede49843ed7e5c000c08b4f55c50897004bf90). Hopefully it answers your question, LMK if not!

Reply:
The explanation is great!